core/src/main/java/org/bitcoinj/base/BitcoinNetwork.java

```diff
@@ -76,17 +77,49 @@ public Coin maxMoney() {
         return MAX_MONEY;
     }
 
+    /**
+     * Get the {@code BitcoinNetwork} from a <i>validated</i> name String
+     * @param nameString A name string (e.g. "MAIN", "TEST", "REGTEST")
```

Comment:
I think here I'd prefer to see `SIGNET` as an example string, as it is more relevant than `REGTEST` these days.

Reply:
Fixed.